### PR TITLE
Made bot auto respawning togglable

### PR DESCRIPTION
**Pull request by Averagess**
Implements #2971

### CODE REVIEW

File: lib/loader.js [hunk 1]
```diff
@@ -118,6 +119,7 @@ function createBot (options = {}) {
       throw new Error(`Version ${version.minecraftVersion} is not supported. Latest supported version is ${latestTestedVersion}.`)
     }
 
+    bot.respawn = options.respawn
```
Comment (extremeheat) [inline lib/loader.js:122]: Why?

File: lib/plugins/health.js [hunk 2]
```diff
@@ -24,6 +24,7 @@ function inject (bot) {
         bot.isAlive = false
         bot.emit('death')
       }
+      if (!bot.respawn) return
```
Comment (extremeheat) [inline lib/plugins/health.js:27]: What are the side effects of this?

**Review by extremeheat — CHANGES_REQUESTED** [review]

File: lib/loader.js (hunk 1 above)
Comment (Averagess) [reply lib/loader.js]: Removed this

File: lib/plugins/health.js (hunk 2 above)
Comment (Averagess) [reply lib/plugins/health.js]: None that i know, please tell me if there are some

Comment (Jovan-04) [reply lib/plugins/health.js]: > What are the side effects of this?
> None that i know, please tell me if there are some

it appears that there's no way to manually trigger the bot respawning aside from calling `bot._client.write('client_command', { payload: 0 })`

perhaps a `bot.respawn()` command would be useful?

**Pull request merged**